Resources/Prototypes/Entities/Objects/Decoration/present.yml

```diff
@@ -381,7 +381,38 @@
   - type: SpawnItemsOnUse
     items:
       - id: PresentTrash
-      - id: Ash # No coal yet!
```

Comment:
Just remove the ash present, we got coal (ash used to be a replacement for coal before we got coal)

Reply:
Sure, I threw a migration in for ya